specification/eventgrid/Azure.Messaging.EventGrid.SystemEvents/Microsoft.Communication/AzureCommunicationServices.tsp

```diff
@@ -947,18 +997,110 @@ model AcsChatMessageEventBaseProperties extends AcsChatEventBaseProperties {
   composeTime: utcDateTime;
 
   /** The type of the message */
-  type?: string;
+  type: string;
 
   /** The version of the message */
-  version?: int64;
+  @madeRequired(ServiceApiVersions.v2024_01_01)
+  version: int64;
+}
+
+/** Schema of common properties of all participant events */
+#suppress "@azure-tools/typespec-azure-core/composition-over-inheritance" "Maintain compatibility with existing models."
+model AcsCallParticipantEventProperties extends AcsCallingEventProperties {
+  /** The user of the call participant */
+  user?: AcsCallParticipantProperties;
+
+  /** The display name of the participant. */
+  displayName?: string;
+
+  /** The id of the participant. */
+  participantId?: string;
+
+  /** The user agent of the participant. */
+  userAgent?: string;
+}
+
+/** Schema of common properties of all participant event user */
+#suppress "@azure-tools/typespec-azure-core/composition-over-inheritance" "Maintain compatibility with existing models."
+model AcsCallParticipantProperties {
+  /** The communication identifier of the participant user */
+  communicationIdentifier?: CommunicationIdentifierModel;
+
+  /** The role of the participant */
+  role?: string;
+}
+
+/** Schema of common properties of all calling events */
+#suppress "@azure-tools/typespec-azure-core/composition-over-inheritance" "Maintain compatibility with existing models."
+model AcsCallingEventProperties {
+  /** The call participant who initiated the call. */
+  startedBy: AcsCallParticipantProperties;
+
+  /** The call id of the server */
+  serverCallId: string;
+
+  /** The group metadata */
+  group?: AcsCallGroupProperties;
+
+  /** The room metadata */
+  room?: AcsCallRoomProperties;
+
+  /** Is two-party in calling event. */
+  isTwoParty?: boolean;
+
+  /** The correlationId of calling event */
+  correlationId: string;
+
+  /** Is the calling event a room call. */
+  isRoomsCall?: boolean;
+}
+
+/** Schema of calling event group properties */
+#suppress "@azure-tools/typespec-azure-core/composition-over-inheritance" "Maintain compatibility with existing models."
+model AcsCallGroupProperties {
+  /** Group Id.*/
+  id?: string;
+}
+
+/** Schema of calling event room properties */
+#suppress "@azure-tools/typespec-azure-core/composition-over-inheritance" "Maintain compatibility with existing models."
+model AcsCallRoomProperties {
+  /** Room Id.*/
+  id?: string;
+}
+
+/** Schema of calling event endedby properties */
+#suppress "@azure-tools/typespec-azure-core/composition-over-inheritance" "Maintain compatibility with existing models."
+model AcsCallEndedByProperties {
+  /** The communication identifier of the call ended by */
+  communicationIdentifier: CommunicationIdentifierModel;
+
+  /** The type of call ended by. */
+  type: AcsCallEndedByKind;
```

Comment:
rename to kind

Reply:
We probably can't change the wire payload, btu we can apply a rename for all languages in https://github.com/Azure/azure-rest-api-specs/blob/main/specification/eventgrid/Azure.Messaging.EventGrid.SystemEvents/propertyNameOverride.tsp